packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -424,7 +403,7 @@ Future<void> _writeAndroidPluginRegistrant(FlutterProject project, List<Plugin>
       }
       templateContent = _androidPluginRegistryTemplateNewEmbedding;
     break;
-    case '1':
+    case AndroidEmbeddingVersion.v1:
       templateContent = _androidPluginRegistryTemplateOldEmbedding;
     break;
     default:
```

Comment:
nit: since it's now using an enum, this `switch` doesn't need a default case anymore.

Reply:
oh right, thanks